packages/gatsby-source-wordpress/README.md

```diff
@@ -773,9 +773,11 @@ exports.createPages = ({ graphql, actions }) => {
         // Create Page pages.
         const pageTemplate = path.resolve("./src/templates/page.js")
         // We want to create a detailed page for each
-        // page node. We'll just use the WordPress Slug for the slug.
+        // page node. We'll use the WordPress link for the slug to preserve url structure.
         // The Page ID is prefixed with 'PAGE_'
         _.each(result.data.allWordpressPage.edges, edge => {
+          const link = new URL(edge.node.link)
```

Comment:
we need to add `link` field to graphql query few lines above (and few lines below) and can also delete `slug` field from query